src/stepfunctions/steps/choice_rule.py

```diff
@@ -376,6 +592,122 @@ def TimestampGreaterThanEquals(cls, variable, value):
+
+    @classmethod
+    def IsNull(cls, variable, value):
+        """
+        Creates a rule with the `IsNull` operator.
+
+        Args:
+            variable (str): Path to the variable to compare.
+            value (bool): Whether the value at `variable` is equal to the JSON literal null or not.
+
+        Returns:
+            Rule: Rule with `IsNull` operator.
+        """
+        return Rule(variable, 'IsNull', value)
+
+    @classmethod
+    def IsPresent(cls, variable, value):
+        """
+        Creates a rule with the `IsPresent` operator.
+
+        Args:
+            variable (str): Path to the variable to compare.
+            value (bool): Whether a field at `variable` exists in the input or not.
+
+        Returns:
+            Rule: Rule with `IsPresent` operator.
+        """
+        return Rule(variable, 'IsPresent', value)
+
+    @classmethod
+    def IsString(cls, variable, value):
+        """
+        Creates a rule with the `IsString` operator.
+
+        Args:
+            variable (str): Path to the variable to compare.
+            value (bool): Whether the value at `variable` is a string or not.
+
+        Returns:
+            Rule: Rule with `IsString` operator.
+        """
+        return Rule(variable, 'IsString', value)
+
+    @classmethod
+    def IsNumeric(cls, variable, value):
+        """
+        Creates a rule with the `IsNumeric` operator.
+
+        Args:
+            variable (str): Path to the variable to compare.
+            value (bool): Whether the value at `variable` is a number or not.
+
+        Returns:
+            Rule: Rule with `IsNumeric` operator.
+        """
+        return Rule(variable, 'IsNumeric', value)
+
+    @classmethod
+    def IsTimestamp(cls, variable, value):
+        """
+        Creates a rule with the `IsTimestamp` operator.
+
+        Args:
+            variable (str): Path to the variable to compare.
+            value (bool): Whether the value at `variable` is a timestamp or not.
+
+        Returns:
+            Rule: Rule with `IsTimestamp` operator.
+        """
+        return Rule(variable, 'IsTimestamp', value)
+
+    @classmethod
+    def IsBoolean(cls, variable, value):
+        """
+        Creates a rule with the `IsBoolean` operator.
+
+        Args:
+            variable (str): Path to the variable to compare.
+            value (bool): Whether the value at `variable` is a boolean or not.
+
+        Returns:
+            Rule: Rule with `IsBoolean` operator.
+        """
+        return Rule(variable, 'IsBoolean', value)
+
+    @classmethod
+    def StringMatches(cls, variable, value):
+        """
+        Creates a rule with the `StringMatches` operator.
+
+        Args:
+            variable (str): Path to the variable to compare.
+            value (str): A string pattern that may contain one or more `*` characters to compare the value at `variable` to.
+            The `*` character can be escaped using two backslashes.
```

Comment:
Not sure if this docstring will render this correctly if the newlines aren't indented